src/site-header/SiteHeader.tsx

```diff
@@ -18,21 +18,6 @@ const Nav = styled.nav`
   justify-content: space-between;
 `;
 
-const LogoContainer = styled.div`
-  align-items: flex-end;
-  color: ${Colors.red};
-  display: flex;
-  font-family: "Poppins", sans-serif;
-  font-size: 13px;
-  font-weight: 600;
-  letter-spacing: -0.05em;
-  text-transform: uppercase;
-`;
```

Comment:
I believe I can remove this container and CSS since we removed the text. See my comment for visual comparison